src/main/kotlin/org/ozwillo/dcimporter/service/ConnectorsService.kt

```diff
@@ -81,6 +81,27 @@ class ConnectorsService(private val businessAppConfigurationRepository: Business
             }
     }
 
+    fun clone(id: String, siret: String): Mono<HttpStatus>{
+        val fallback: Mono<BusinessAppConfiguration> =
+                Mono.error(EmptyException("No connector found with id $id"))
+
+        return businessAppConfigurationRepository.findById(id)
+            .switchIfEmpty(fallback)
+            .flatMap { existingConnector ->
+                val businessAppConfiguration =
+                    BusinessAppConfiguration(
+                        baseUrl = existingConnector.baseUrl,
+                        organizationSiret = siret,
+                        instanceId = existingConnector.instanceId,
+                        login = existingConnector.login,
```

Comment:
we should avoid copying credential info to enforce editors to choose fresh ones each time they declare a new connector

Reply:
I'm not sure to understand here. I thought that the cloning logic consists of duplicate the same infos for a new organization. If we ask for new credential infos what is the difference with a classic update form then ?

Reply:
it is not an update, since you create a new connector from some existing info.

but I don't want editors to use the same credential info for **all** their connectors (if one leaks, they all leak).

Reply:
Aaah ok, I understood. I changed it, now users have to enter their own credential infos in the cloning form (or let them empty if connector don't need it). Application name and application base url are still taken from the existing connector to avoid errors.